src/Identity/Core/src/Microsoft.AspNetCore.Identity.csproj

```diff
@@ -31,7 +31,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <!-- TODO: Re-enable RDG once https://github.com/dotnet/aspnetcore/issues/47918 is fixed. -->
-    <!--<ProjectReference Include="$(RepoRoot)/src/Http/Http.Extensions/gen/Microsoft.AspNetCore.Http.RequestDelegateGenerator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />-->
+    <ProjectReference Include="$(RepoRoot)/src/Http/Http.Extensions/gen/Microsoft.AspNetCore.Http.RequestDelegateGenerator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```

Comment:
We should also be able to remove `[RequiresDynamicCode]` from `MapIdentityApi<TUser>()`.

https://github.com/dotnet/aspnetcore/blob/0c8a569b9b5880ce54dc65c29647df0c99e775a9/src/Identity/Core/src/IdentityApiEndpointRouteBuilderExtensions.cs#L31-L32